gr-blocks/lib/correctiq_auto_impl.cc

```diff
@@ -123,11 +123,13 @@ float correctiq_auto_impl::get_gain() const { return d_gain; }
 
 void correctiq_auto_impl::set_freq(double new_value)
 {
+    d_freq = new_value;
```

Comment:
What's the purpose of these?

Reply:
I don't really know. You'll have to ask @ghostop14. I guess it's some convenience placeholder.

When I was repairing the block, it seemed like setting the variables had been omitted, so I put those lines in.

Reply:
Hi folks, I just looked at the current implementation and I think as long as the synchronized flag gets set to false, it can be removed.  Originally I probably checked that the value of freq there actually changed to save an update, but doesn't look like I'm doing that anymore.  So looks safe to pull it.  Same looks true for d_gain.

Reply:
But don't you want the values returned by get_freq() and get_gain to be what you set them to with set_freq() and set_gain()?

Reply:
True.  So maybe just leave it as-is.